packages/superset-ui-chart/src/query/processFilters.ts

```diff
@@ -0,0 +1,51 @@
+import { ChartFormData } from '../types/ChartFormData';
+import { QueryObjectFilterClause } from '../types/Query';
+import { isSimpleAdhocFilter } from '../types/Filter';
+import convertFilter from './convertFilter';
+
+/** Logic formerly in viz.py's process_query_filters */
+export default function processFilters(formData: ChartFormData) {
+  // TODO: Implement
+  // utils.convert_legacy_filters_into_adhoc(self.form_data)
+
+  // TODO: Implement
+  // merge_extra_filters(self.form_data)
+
+  // utils.split_adhoc_filters_into_base_filters(self.form_data)
```

Comment:
is this just indicating what the logic below mapped to? (i.e., it's not part of the TODOs above) should we remove it or keep for context?

Reply:
Will remove or change to simple sentence. Seem not necessary any more